conf/realm.conf.defaults

```diff
@@ -1,14 +1,14 @@
-[DEFAULT]
+[1 DEFAULT]
```

Comment:
I think we discussed it but for the first tenant, the ID wouldn't be necessary (it would be implicit that its tenant 1 if there is no tenant ID there)

That would keep the configuration simpler and would allow for backward compatibility with our current admin

Reply:
I would prefer to keep it consistent and not have special cases.
With that said will need to add a script to handle the upgrade process.